docs/run-platform-server/ingestion.mdx

```diff
@@ -5,26 +5,33 @@ sidebar_position: 45
 
 import { CodeExample } from "@site/src/components/CodeExample";
 
-Horizon provides access to both current and historical state on the Stellar network through a process called **ingestion**.
-
-Horizon provides most of its utility through ingested data, and your Horizon server can be configured to listen for and ingest transaction results from the Stellar network. Ingestion enables API access to both current (e.g. someone's balance) and historical state (e.g. someone's transaction history).
+Horizon API provides most of its utility through ingested data, and your Horizon server can be configured to listen for and ingest transaction results from the Stellar network. Ingestion enables API access to both current (e.g. someone's balance) and historical state (e.g. someone's transaction history).
 
 ## Ingestion Types
 
 There are two primary ingestion use-cases for Horizon operations:
 
-- ingesting **live** data to stay up to date with the latest, real-time changes to the Stellar network, and
-- ingesting **historical** data to peek how the Stellar ledger has changed over time
+- ingesting **live** data to stay up to date with the latest ledgers from the network, accumulating a sliding window of aged ledgers, and
+- ingesting **historical** data to retroactively add network data from a time range in the past to the database.
+
+## Determine storage space
+
+You should think carefully about the historical timeframe of ingested data you'd like to retain on Horizon's database. The storage requirements for every transaction from Stellar network are substantial and grow unbounded, increasing storage used by the Horizon database well into many terabytes and is an unsustainable deployment. Most organizations and operators only need a recent fraction of the historical data, to fit their use case. To Achieve minimal storage footprint, we recommend the following best practices:
+
+- use **live** ingestion. only use and depend on **historical** ingestion in limited exceptional cases.
+- use [ingestion filter rules](./ingestion-filtering.mdx) based on accounts and assets related to your application space.
+- use temporal limits, if no filter rules are applied, we recommend limiting historical retention of ingested data to a sliding window of 1 month(`HISTORY_RETENTION_COUNT=518400`) and this is the [default set by Horizon](./configuring.mdx#ingestion-role-instance). This is based on metrics and trend analysis of our own API servers, we have seen most requests are for near term data.
+  - if not using filter rules and want to access historical data more than 1 month old, we recommend not using Horizon ingestion and its database for this purpose, instead consider usage of [Stellar Hubble Data Warehouse](https://www.stellar.org/developers-blog/try-our-new-analytics-dataset?locale=en).
```

Comment:
```suggestion
You should think carefully about the historical timeframe of ingested data you'd like to retain in Horizon's database. The storage requirements for transactions on the Stellar network are substantial and are growing unbounded over time. This is something that you may need to continually monitor and reevaluate as the network continues to grow. We have found that most organizations need only a small fraction of recent historical data to satisfy their use cases. Through analyzing traffic patterns on SDF's Horizon instance, we see that most requests are for very recent data. 

To keep your storage footprint small, we recommend the following:

- use **live** ingestion only use and depend on **historical** ingestion in limited exceptional cases
- we recommend limiting historical retention of ingested data to a sliding window of 1 month (`HISTORY_RETENTION_COUNT=518400`) and this is the [default set by Horizon](./configuring.mdx#ingestion-role-instance)
- if you cannot limit your history retention window, use [ingestion filter rules](./ingestion-filtering.mdx) to include only accounts and assets relevant to your application
- if you cannot limit your history retention window, and cannot use filter rules, we recommend considering [Stellar Hubble Data Warehouse](https://www.stellar.org/developers-blog/try-our-new-analytics-dataset?locale=en) for any historical data
```

Reply:
blended your suggestions in - https://github.com/stellar/stellar-docs/pull/168/commits/948494fff26bec22a0b4b01f636c26f393722265